bson/src/test/unit/org/bson/codecs/pojo/PojoCustomTest.java

```diff
@@ -573,6 +578,17 @@ public void testInvalidGetterAndSetterModelDecoding() {
         decodingShouldFail(getCodec(InvalidGetterAndSetterModel.class), "{'integerField': 42, 'stringField': 'myString'}");
     }
 
+    @Test(expected = CodecConfigurationException.class)
+    public void testExceptionWithWildcardList() {
+        roundTrip(getPojoCodecProviderBuilder(InterfaceBasedModel.class, AbstractInterfaceModel.class,
+                ConcreteAndNestedAbstractInterfaceModel.class, ConcreteStandAloneAbstractInterfaceModel.class),
+                new ConcreteAndNestedAbstractInterfaceModel("A",
+                        singletonList(new ConcreteStandAloneAbstractInterfaceModel("B"))),
+                "{'_t': 'org.bson.codecs.pojo.entities.ConcreteAndNestedAbstractInterfaceModel', 'name': 'A', "
+                        + "  'wildcardList': [{'_t': 'org.bson.codecs.pojo.entities.ConcreteStandAloneAbstractInterfaceModel', "
```

Comment:
I'm still confused as to why when we have a discriminator this should be expected to fail.  Is the `classModel.useDiscriminator` false in this case?  

We should have a test for where its true it can roundtrip when using the discriminator and it fails when its not present and / or the `classModel.useDiscriminator=false`

Reply:
Yes, `classModel.useDiscriminator` is false for the wildcard list. I do not see a way to create a test case where the discriminator can be true when using wildcards. Suggestions welcome.

Reply:
I have updated the test to use a discriminator for the list of interfaces and the test now succeeds.